package.json

```diff
@@ -95,8 +95,7 @@
       "^.+\\.(t|j)s$": "ts-jest"
     },
     "roots": [
-      "./src",
-      "./migrations"
```

Comment:
I would suggest we only remove test coverage of now-deprecated migrations when we remove the v1 database integration.

Reply:
The plan was to remove the integration, but that should be done in a separate PR, especially since other modules are now using it.
https://github.com/safe-global/safe-client-gateway/commit/2ea0e5d8a752fd31cff745ff3490d8fe8134ddc7